src/flint/flint_base/flint_base.pyx

```diff
@@ -190,3 +296,7 @@ cdef class flint_mat(flint_elem):
 
     # supports mpmath conversions
     tolist = table
+
+
+cdef class flint_rational_function(flint_elem):
+    pass
```

Comment:
I would remove this for now unless you intend to expand it significantly in this PR.

Reply:
That's fair enough, this work should come eventually, but maybe not in this PR.